oaknational/Oak-Web-Application
# chore(storybook): default to intro page and fix viewMode docs

## Description

- puts "Introduction" first (and only) in order array, meaning it's the first page landed on
- fixed the viewMode=docs usage so that each link will click to the docs version of the story


## How to test

1. Go to {cloud storybok link}
2. Should see intro page first
3. CLick a story
4. Should see it in docs mode!


## Checklist
